### PR TITLE
AWS Quickstarter Bug fixes

### LINKED ISSUE #629: AWS Plain QS - AWS Accounts are mapped to the same variables
**Describe the bug**
AWS account information (TESTING/D/Q/P) is stored in the OC projects secret store. Mapping to TESTING,D,Q & P is handled via separate environment yaml files. Currently these files are created during the QS provisioning process from the same template file. Which means that each environment points to the same account information: 
https://github.com/opendevstack/ods-quickstarters/blob/38814dfaa64d93f49930b2ab089f54e806f0862c/inf-terraform-aws/environment.yml.template#L4 
In this case always to D.

**To Reproduce**
Deploy a QS into a project and verify the content of the yaml files in ./environments/. 

**Expected behavior**
Each yaml file shall contain the correct variable mapping (eg access-key):
dev.yml:
`  
key: @project_id@-cd-aws-access-key-id-dev
`
test.yml:
`  
key: @project_id@-cd-aws-access-key-id-test
`
prod.yml:
`  
key: @project_id@-cd-aws-access-key-id-prod
`

**Affected version (please complete the following information):**
 - AWS Plain QS 3.x

**Additional context**
It's not a critical bug, since the user can fix these files on his own. But since the bug is easy to fix we can improve the user experience.

### PR DESCRIPTION
**Pull request by nichtraunzer**
Fixes #629 #633

### CODE REVIEW

**Comment by metmajer** [comment]
Looks nice @nichtraunzer, integration tests are failing again though @clemensutschig. I am already AFK, let's put some of our people on this problem next week please.

File: inf-terraform-aws/dev.yml.template [hunk 1]
```diff
@@ -4,5 +4,4 @@ credentials:
   key: @project_id@-cd-aws-access-key-id-dev
   secret: @project_id@-cd-aws-secret-access-key-dev
 
-# put your account id in here with double quotes           
-account: "nnnnnnnnn"
+account: "<your_aws_account_id"
```
Comment (tbugfinder) [inline inf-terraform-aws/dev.yml.template:7]: looks like `<` could be removed or an ending `>` should be included.

**Review by nichtraunzer — COMMENTED** [review]
Thanks for the finding 👍

**Comment by nichtraunzer** [comment]
> Looks nice @nichtraunzer, integration tests are failing again though @clemensutschig. I am already AFK, let's put some of our people on this problem next week please.

It seems some "magic" happened during the weekend :) - integration tests succeeded

**Review by metmajer — APPROVED** [review]

**Pull request merged**